planner/core/integration_test.go

```diff
@@ -903,14 +903,6 @@ func TestIndexMergePartialScansClusteredIndex(t *testing.T) {
 		condition string
 		expected  []string
 	}{
-		{
-			// 3 table scans
-			"a < 2 or a < 10 or a > 11", []string{"1", "100"},
-		},
-		{
-			// 3 index scans
-			"c < 10 or c < 11 or c > 50", []string{"1", "10", "100"},
-		},
```

Comment:
Should these return "TableFullScan" or something else? Maybe the cases struct could be extended with matching MustHavePlan/MustNotHavePlan?

Reply:
Yes, it's `TableFullScan`. Not sure it should be checked in this test or not.